### ISSUE TITLE
mixed precision for lnms

### ISSUE THREAD

**Issue by Matze99** [DigitalSlideArchive/HistomicsDetect#63]
The one thing is to make sure your network output is fp32 - you can add a linear activation to each output as they show in the TF page

outputs = layers.Activation('softmax', dtype='float32', name='predictions')(x)
change 'softmax' to linear and it's basically just a casting operation

https://docs.nvidia.com/deeplearning/performance/mixed-precision-training/index.html

https://www.tensorflow.org/guide/mixed_precision
